### PR TITLE
Update AI Teaching Assistant landing page for launch

### PR DESCRIPTION
**Pull request by etaderhold**
Post-launch updates to AI Teaching Assistant landing page.

Updates header image to reflect Best of STEM award, adds a new section for an intro video, adds another new section for links to news articles, and updates a couple of the FAQ items.

### Top of page (before)
image

### Top of page (after):
* Updated top image
* Added new "get started" section
* Reversed text/image order on next two sections.
image

### New "In the news" section
image

### Two Updated FAQs (before)
image

### FAQs (after)
image

## Links
* [Figma](https://www.figma.com/design/5Pcxl4tcFgTeKDHZJUAQ5p/AI-TA-Launch?node-id=2778-1406&node-type=section&t=vsprxF64NB3eiGTJ-0)
* [JIRA](https://codedotorg.atlassian.net/browse/AITT-785)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by tess323** [comment]
Thank you!

File: pegasus/sites.v3/code.org/public/ai/teaching-assistant.haml [hunk 1]
```diff
@@ -13,24 +13,70 @@ theme: responsive_full_width
...
+        .action-block.action-block--one-col.flex-space-between
+          .content-wrapper
+            %p.overline
+              =hoc_s("ai_ta_page.in_the_news.#{news_item[:slug]}_label")
+            %h3
+              =hoc_s("ai_ta_page.in_the_news.#{news_item[:slug]}_headline")
+            %img{src: "/images/ai/page/teaching-assistant/#{news_item[:slug]}-image.png", alt: ""}
+          .content-footer
+            %a.link-button{href: news_item[:url]}
+              =hoc_s("ai_ta_page.in_the_news.#{news_item[:slug]}_button")
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/ai/teaching-assistant.haml]: Can you move this, and lines 53-63 into a separate view in [this directory](https://github.com/code-dot-org/code-dot-org/tree/staging/pegasus/sites.v3/code.org/views/ai_teaching_assistant)? You can then import it similarly to what's on line 84.

File: pegasus/sites.v3/code.org/public/ai/teaching-assistant.haml [hunk 2]
```diff
@@ -13,24 +13,70 @@ theme: responsive_full_width
...
+        %ol
+          %li=hoc_s("ai_ta_page.get_started.watch_demo")
+          %li=hoc_s("ai_ta_page.get_started.assign_csd_3")
+          %li
+            %a{href: "https://form.jotform.com/240863841362155"}
+              = hoc_s("ai_ta_page.get_started.become_verified")
+          %li=hoc_s("ai_ta_page.get_started.use_rubric")
+        .content-footer
+          %a.link-button{href: "https://studio.code.org/s/csd3-2024"}
+            = hoc_s("ai_ta_page.get_started.use_ta")
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/ai/teaching-assistant.haml]: Nit: move this into a view too. This makes it easier to update sections if/when the time comes.

**Review by kelbyhawn — COMMENTED** [review]
Thank you for doing this! 

I left a couple clean up comments, but can you also post a screenshot of the updates you made to the page so it's easier to see what changed? I usually do a before and after or a full page screenshot with arrows pointing to the new stuff. ([see example](https://github.com/code-dot-org/code-dot-org/pull/61023))

**Comment by etaderhold** [comment]
Thanks for the review! I split up the file and added screenshots.

File: pegasus/sites.v3/code.org/views/ai_teaching_assistant/ai_ta_in_the_news.haml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      slug: "education_week",
+      url: "https://www.edweek.org/technology/this-ai-tool-cut-one-teachers-grading-time-in-half-how-it-works/2024/04",
+    },
+    {
+      slug: "blog",
+      url: "https://codeorg.medium.com/code-org-wins-2024-best-of-stem-award-155e55bde866",
+    },
+  ]
+
+%h2.white=hoc_s("ai_ta_page.in_the_news.header")
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/ai_teaching_assistant/ai_ta_in_the_news.haml:13]: Nit: you can keep this in the main file, but this is def a no biggie!

File: pegasus/sites.v3/code.org/public/images/ai/page/teaching-assistant/education_week-image.png
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/images/ai/page/teaching-assistant/education_week-image.png:1]: This image should be the same height as the Best of Stem image so the boxes look more even:
education_week-image

**Review by kelbyhawn — APPROVED** [review]
Two more nits, but looks good. Thanks for including the screenshots!

**Review by kelbyhawn — APPROVED** [review]

**Pull request merged**